www/js/app.js

```diff
@@ -631,31 +631,36 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         apiStatusPanel.classList.remove('card-success', 'card-warning', 'card-danger');
         var apiPanelClass = 'card-success';
         if (isMessageChannelAvailable()) {
-            $('#messageChannelStatus').html("MessageChannel API available");
-            $('#messageChannelStatus').removeClass("apiAvailable apiUnavailable")
-                    .addClass("apiAvailable");
+            let messageChannelStatus = document.getElementById('messageChannelStatus');
```

Comment:
Instead of defining this here and then again in line 640, it would be better to define it once above this `if` block.

Reply:
Yes, this makes sense. I shall correct this.